### PR TITLE
JIRA:VZ-2461, VZ-2543 create oam kubernetes runtime cluster role

### PR DESCRIPTION
**Pull request by jmaron99**
# Description

- Defines the oam kubernetes runtime cluster role
- removes deprecated cluster role from application operator scripts (these scripts should be phased out - they have significantly diverged from the standard installation)

Additionally:

- Fixes VZ-2543 by leveraging verrazzano ingress rather than rancher ingress for determining domain
- Adds multi cluster resources to AT cluster dumps

Associated builds:
OCI DNS:  https://build.verrazzano.io/job/verrazzano-new-oci-dns-acceptance-tests/job/jmaron%252FVZ-2461/ (failure is in the post phase due to terraform issue - tests passed)
Multi-cluster AT:  https://build.verrazzano.io/job/verrazzano-multi-cluster-acceptance-tests/job/jmaron%252FVZ-2461/12/
VZ AT:  https://build.verrazzano.io/job/verrazzano/job/jmaron%252FVZ-2461/19/

Fixes VZ-2461, VZ-2543

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [x] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: application-operator/controllers/ingresstrait/ingresstrait_controller_test.go [hunk 1]
```diff
@@ -163,17 +164,17 @@ func TestSuccessfullyCreateNewIngress(t *testing.T) {
 	mock.EXPECT().
 		Get(gomock.Any(), types.NamespacedName{Namespace: "istio-system", Name: "test-space-myapp-cert"}, gomock.Not(gomock.Nil())).
 		Return(k8serrors.NewNotFound(schema.GroupResource{Group: "test-space", Resource: "Certificate"}, "test-space-myapp-cert"))
-	// Expect a call to get the Rancher ingress and return the ingress.
+	// Expect a call to get the verrazzano ingress and return the ingress.
 	mock.EXPECT().
-		Get(gomock.Any(), types.NamespacedName{Namespace: "cattle-system", Name: "rancher"}, gomock.Not(gomock.Nil())).
+		Get(gomock.Any(), types.NamespacedName{Namespace: constants.VerrazzanoSystemNamespace, Name: "verrazzano-ingress"}, gomock.Not(gomock.Nil())).
```
Comment (mgianatagh) [inline application-operator/controllers/ingresstrait/ingresstrait_controller_test.go:169]: There is a constant for the Verrazzano-ingress: constants.VzConsoleIngress

File: application-operator/controllers/ingresstrait/ingresstrait_controller_test.go [hunk 2]
```diff
@@ -850,17 +851,17 @@ func TestSuccessfullyCreateNewIngressForVerrazzanoWorkload(t *testing.T) {
 		DoAndReturn(func(ctx context.Context, certificate *certapiv1alpha2.Certificate, opts ...client.CreateOption) error {
 			return nil
 		})
-	// Expect a call to get the Rancher ingress and return the ingress.
+	// Expect a call to get the verrazzano ingress and return the ingress.
```
Comment (mgianatagh) [inline application-operator/controllers/ingresstrait/ingresstrait_controller_test.go:854]: Should these references be Verrazzano be capitalized?

File: application-operator/controllers/ingresstrait/ingresstrait_controller_test.go [hunk 3]
```diff
@@ -2446,14 +2447,14 @@ func appendAsUnstructured(list *unstructured.UnstructuredList, object interface{
...
 
-// newRancherIngress creates a new Ranger Ingress with the provided IP address.
-func newRancherIngress(ipAddress string) *k8net.Ingress {
+// newVerrazzanoIngress creates a new Ranger Ingress with the provided IP address.
+func newVerrazzanoIngress(ipAddress string) *k8net.Ingress {
 	rangerIngress := k8net.Ingress{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "rancher",
-			Namespace: "cattle-system",
+			Name:      "verrazzano-ingress",
```
Comment (mgianatagh) [inline application-operator/controllers/ingresstrait/ingresstrait_controller_test.go:2454]: Use constants for name and namespace

**Review by mgianatagh — APPROVED** [review]

**Review by wmhopkins — APPROVED** [review]
Looks good to me, if Mike G.'s comments are addressed.

**Pull request merged**